Update static_model.json for KevinHall_FINCH

M-Ghosh9 opened the pull request:
Added new food and updated all placeholders to allow values for region, ethnicity and income models. 

All the values need to be updated.

jzhu20 commented on KevinHall_FINCH/static_model.json:
```diff
@@ -24,26 +168,47 @@
-                "Over18": 0.298055376100666,
-                "Region": -0.383372795038086,
+                "Age": 0.005,
+                "England": 0.1,
```
There should be age_squared as well.

M-Ghosh9 replied:
Okay. I've done this in my local repository. I'll update here too.

jzhu20 commented on KevinHall_FINCH/static_model.json:
```diff
@@ -15,6 +15,150 @@
+        {
+            "Name": "Under18",
+            "Female": {
+                "England": {
```
The basic idea of EthnicityPrevalence is correct. But just as we said, there won't be different prevalence by region.

M-Ghosh9 commented on KevinHall_FINCH/static_model.json:
```diff
@@ -15,6 +15,150 @@
+        }
+    ],
     "IncomeModels": {
         "Low": {
```
Yes quartiles are assigned after continuous income is assigned.

jzhu20 commented on KevinHall_FINCH/static_model.json:
```diff
@@ -1,51 +1,122 @@
+    "$schema": "https://raw.githubusercontent.com/imperialCHEPI/healthgps/main/schemas/v2/config/models/staticlinear.json",
     "ModelName": "StaticLinear",
     "InformationSpeed": 0.1,
     "PhysicalActivityStdDev": 0.06,
```
line 5 of physical activity standard deviation overlaps with line 119 of physical activity standard deviation in the model. Which parameter is really used?

M-Ghosh9 replied:
The static_model.json value will be used unless that is missing, it will use the default value in the codebase

jzhu20 replied:
I'm confused. Maybe I rephrase my comment - I have standard deviation from physical activity model. Where should I put the value of sd, line 5 or line 119? Hope it's clear now.

M-Ghosh9 replied:
You can add the same value in both the places. This was in the original schema and I didn't modify it. So you can just put the same value in both places- the code will take the latter value.

jzhu20 commented on KevinHall_FINCH/static_model.json:
```diff
@@ -1,51 +1,122 @@
+                "Asian": 0.0,
+                "Black": 0.0,
+                "Others": 0.0,
                 "Sector": 0.0
```
Income variable is missing in physical activity models.

M-Ghosh9 replied:
Thanks for pointing in out. Missed it

jzhu20 replied:
I still don't see income here.

jzhu20 reviewed (commented):
Some comments applicable to all the models:

1. Age2 needs to be added in all the policy models. Can we add EnergyIntake in all policy logcoefficients, e.g. lines 155-166?
2. I wonder the values of "Gender" in the models. Is it 1-Male, 2-Female? If so, is it possible to make it 1-Female, 0-Male somehow? Because the coefficient is for female. I am very happy to discuss solutions if needed.
3. Six risk factors are missing in risk factor models: fibre, totalsugar, addedsugar, saturatedfat, polyunsaturatedfattyacid, monounsaturatedfat. All the risk factors are estimated with risk factor models.

M-Ghosh9 commented:
Addressing your comments-

1. Added Age 2 in all policy models
2. Added EnergyIntake to all the Policy Log Coefficients (only)
3. The original gender in the model is 1 Male and 0 female. I've swapped them now to have 1 female and 0 male
4. Apologies for the missing risk factors, they are updated now

jzhu20 reviewed (approved):
Looks good to me now.
